File: BladeQuest/src/bladequest/actions/actResetGame.java
package bladequest.actions;

import bladequest.world.Global;

public class actResetGame extends Action {
	
	public actResetGame()
	{
		super();
	}
	
	@Override
	public void run()
	{
		Global.restartGame();
	}
	
	@Override
	public boolean isDone()
	{
		return true;
	}

}

File: BladeCraft/BladeCraft/Classes/Objects/Actions/actResetGame.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.IO;

namespace BladeCraft.Classes.Objects.Actions
{
    public class actResetGame : Action
    {
        public actResetGame() { actionType = type.RestartGame; }
        public actResetGame(actResetGame other) { actionType = type.RestartGame; }

        public override void write(StreamWriter writer) 
        {
            writer.WriteLine("action resetgame");

        }
    }
}
